### PR TITLE
Add a skip to main content link as the first tab index in every page

### PR DESCRIPTION
**Pull request by wheelsandcogs**
~waiting on a bump of frontend-toolkit after https://github.com/UKHomeOffice/frontend-toolkit/pull/37~

### CODE REVIEW

File: ui/components/home-office.jsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ class HomeOffice extends React.Component {
         <body>
 
           <header role="banner" className={this.props.propositionHeader ? 'with-proposition' : ''}>
+            <a href={this.props.skipToContentTarget} className="govuk-skip-link">{this.props.skipToContent}</a>
```
Comment (lennym) [inline ui/components/home-office.jsx:27]: This should probably have a default value of a thing that reliably exists.

Comment (wheelsandcogs) [reply ui/components/home-office.jsx:27]: see https://github.com/UKHomeOffice/asl-service/blob/master/ui/components/home-office.jsx#L131-L132

Comment (wheelsandcogs) [reply ui/components/home-office.jsx:27]: Links to https://github.com/UKHomeOffice/asl-service/blob/master/ui/views/base.jsx#L73

Comment (lennym) [reply ui/components/home-office.jsx:27]: Oh, so we theoretically already had this, it was just broken?

Comment (wheelsandcogs) [reply ui/components/home-office.jsx:27]: Parts of it were already there, I assume it was either added in prep for this, or the link was removed at some point, but I couldn't find any evidence when or where.

**Pull request merged**